scripts/generate-sdk/languages/java.sh

```diff
@@ -135,6 +135,12 @@ generate_java_sdk() {
         rm "${SERVICES_FOLDER}/${service}/.openapi-generator-ignore"
         rm -r "${SERVICES_FOLDER}/${service}/.openapi-generator/"
 
+        # If the service has a wait package, move them inside the service folder
+        if [ -d ${sdk_services_backup_dir}/${service}/src/main/java/cloud/stackit/sdk/${service}/wait ]; then
```

Comment:
```suggestion
        if [ -d "${sdk_services_backup_dir}/${service}/src/main/java/cloud/stackit/sdk/${service}/wait" ]; then
```

It's best practice to quote such things in bash :sweat_smile: